collection-scripts/gather_metal3

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Resource List
+resources=()
+
+# Metal3 related Resources
+resources+=(ns/openshift-machine-api ns/openshift-machine-config-operator)
+
+for i in $(/usr/bin/oc get crd | grep metal3 | awk '{print $1}')
```

Comment:
Should this be gathered by related resources?

Reply:
My understanding was that we need to collect this. @etingof please confirm